### ISSUE TITLE
CrossEntropyLoss calculation for target containing class probabilities

### ISSUE THREAD

**Issue by kw01sg** [pytorch/pytorch#73288]
### 📚 The doc issue

Under [CrossEntropyLoss](https://github.com/pytorch/pytorch/blob/bd4902d81f75168bdcdfcd4055f96a0602de97e3/torch/nn/modules/loss.py#L1067) in https://github.com/pytorch/pytorch/blob/master/torch/nn/modules/loss.py, latex for how the loss is calculated for targets with probabilities for each class seems to be written wrongly:

```
.. math::
          \ell(x, y) = L = \{l_1,\dots,l_N\}^\top, \quad
          l_n = - \sum_{c=1}^C w_c \log \frac{\exp(x_{n,c})}{\exp(\sum_{i=1}^C x_{n,i})} y_{n,c}
```

### Suggest a potential alternative/fix

In the denominator, summation and exponential should be swapped

cc @brianjo @mruberry @albanD @jbschlosser @walterddr @kshitij12345

**Comment by george-qi** [comment]
Thanks for pointing this out! Just to confirm, you're recommending the following fix of:

```
.. math::
          \ell(x, y) = L = \{l_1,\dots,l_N\}^\top, \quad
          l_n = - \sum_{c=1}^C w_c \log \frac{\exp(x_{n,c})}{\sum_{i=1}^C \exp(x_{n,i})} y_{n,c}
```

correct?

**Comment by kw01sg** [comment]
Yes, that is correct.

**Comment by jbschlosser** [comment]
Thanks for the report! This was already addressed in #70220 and the fix is present in the [master version of the docs](https://pytorch.org/docs/master/generated/torch.nn.CrossEntropyLoss.html). Closing as completed.
